gravityflow/gravityflow
# Add the "All fields except" choice to the Display Fields setting

This adds a new choice to the Display Fields setting allowing all fields except the selected fields to be displayed on the workflow detail page.

**Testing Instructions**

Check approval and user input steps which use the "all fields" and "selected fields" choices continue to display the same fields as master.

Configure approval and user input steps with the "all fields except" choice selected and one or more fields selected which shouldn't be displayed. Submit the form and then progress through the workflow checking the selected fields are not displayed.
